src/app/shared/utils/play-sound.ts

```diff
@@ -0,0 +1,4 @@
+export const playSound = (sound: string) => {
```

Comment:
create a SoundService on /core and apply this logic to a method, then we could go ahead and create an enum for the input instead of a number so we could know the list of the sound that we have.